Pd 1356 update apps UI ref content (#2972)
truenas/documentation: content/SCALE/SCALEUIReference/Apps/_index.md

## Installed Screen

The **Apps** navigation option opens the **Installed** applications screen.
The screen shows **No Applications Installed** if an application is not installed.

{{< trueimage src="/images/SCALE/Apps/AppsInstalledAppsScreenNoApps.png" alt="Installed Applications Screen No Apps" id="Installed Applications Screen No Apps" >}}

The first time you open the **Installed** applications screen a dialog prompts you to choose the pool for apps to use for storage.
Select the pool from the dropdown list, then click **Save**. This starts the applications service.
If you exit out of this dialog, to set the pool, click [**Settings > Choose Pool**](#choose-a-pool-for-apps-dialog) to select a storage pool for apps.

After setting the apps storage pool, the screen shows <span class="iconify" data-icon="mdi:check-circle" color=#71BF44></span> **Apps Service Running**.

Click **Check Available Apps** or **Discover Apps** to open the **[Discover](#discover-screen)** applications screen to access widgets for applications available in SCALE.

After installing an application, the **Installed** screen populates the **Applications** table.

> When returing to the **Installed** screen, the first application on the list is selected by default.

Reviewer: ```suggestion
When returning to the **Installed** screen, the first application on the list is selected by default.
```

Each application row shows the name, status, and update information for the application.

{{< trueimage src="/images/SCALE/Apps/InstalledAppsScreenWithApps.png" alt="Installed Applications Status" id="Installed Applications Status" >}}

**Search** allows entering the name of an app to locate an installed application.

Selecting the checkbox on an app row shows the **[Bulk Actions](#bulk-actions)** dropdown list.
The checkbox to the left of **Applications** selects all installed apps.

## Settings Menu
**Settings** on the **Installed** applications header displays global options that apply to all applications. 

* **Choose Pool** opens the **[Choose a pool for Apps](#choose-a-pool-for-apps-dialog)** dialog.
* **Unset Pool** shows after setting a pool for applications to use. It opens the **Unset Pool** dialog.
* **Manage Container Images** opens the [**Manage Container Images**](#manage-container-images) screen.

> * **[Train Settings](#train-settings)** opens the **Edit Catalog** screen. Use to add or remove other trains to the **Stable** catalog of applications. <!-- more changes coming in RC -->
> 

Reviewer: ```suggestion
* **[Train Settings](#train-settings)** opens the **Train Settings** screen. Use to add or remove other trains to the **Stable** catalog of applications. <!-- more changes coming in RC -->

```

{{< trueimage src="/images/SCALE/Apps/AppsInstalledAppsSettingOptions.png" alt="Installed Applications Screen Settings" id="Installed Applications Screen Settings" >}}

### Choose a Pool for Apps
**Choose Pool**  opens the **Choose a pool for apps** dialog. The **Pool** dropdown list shows a list of available pools on the system.
**Choose** sets the selected pool for use by applications.

{{< trueimage src="/images/SCALE/Apps/AppsChoosePoolForApps.png" alt="Apps Choose a Pool for Apps" id="Apps Choose a Pool for Apps" >}}

### Train Settings <!-- comment out more image changes expected in RC1. -->  

> **Train Settings** opens the **Edit Catalog** screen.

Reviewer: ```suggestion
**Train Settings** opens the **Train Settings** screen.
```

> {{< trueimage src="/images/SCALE/Apps/AppsEditCatalogScreen.png" alt="Apps Edit Catalog" id="Apps Edit Catalog" >}}

Reviewer: Needs updated image
```suggestion
{{< trueimage src="/images/SCALE/Apps/AppsTrainSettingsScreen.png" alt="Apps Train Settings Screen" id="Apps Train Settings Screen" >}}  
```

**Install** opens the installation wizard for the application.

The bottom of the screen includes widgets for similar applications found in the catalog.

### Application Install or Edit App Wizards
The application **Install *Application*** wizard and **Edit *Application*** screens show the same settings.
The **Edit *Application*** screen opens populated with the current settings for the application.
Settings greyed out are cannot be edited.

The install and edit wizard screens include a navigation panel on the right of the screen that lists and links to the setting sections.
A red triangle with an exclamation point marks the sections with the required settings.
An asterisk marks the required fields in a section.
You can enter a new setting in fields that include a preprogrammed default.

{{< trueimage src="/images/SCALE/Apps/AppsInstallWizardSectionTOC.png" alt="App Installation Wizard ToC" id="App Installation Wizard ToC" >}}

{{< include file="/static/includes/AppsInstallWizardSettings.md" >}}

## Discover Apps Screen
The **Discover** screen displays application widgets for the official TrueNAS **stable** train by default.

> Users can add the **Community** and **Enterprise** train applications on the **[Edit Catalog](#edit-catalog-screen)** screen.

Reviewer: ```suggestion
Users can add the **Community** and **Enterprise** train applications on the **[Train Settings](#train-settings)** screen.
```
